src/ProductConstructionService/Readme.md

```diff
@@ -2,9 +2,11 @@
 
 To run the Product Construction Service locally, set the `ProductConstructionService.AppHost` as Startup Project, and run with F5.
 
-When running locally, the service will attempt to read secrets from the [`ProductConstructionDev`](https://ms.portal.azure.com/#@microsoft.onmicrosoft.com/resource/subscriptions/cab65fc3-d077-467d-931f-3932eabf36d3/resourceGroups/product-construction-service/providers/Microsoft.KeyVault/vaults/ProductConstructionDev/overview) KeyVault, using your Microsoft credentials. If you're having some authentication double check the following:
- - In VS, go to `Tools -> Options -> Azure Service Authentication -> Account Selection` and make sure your corp account is selected
- - Check your environmental variables, you might have `AZURE_TENANT_ID`, `AZURE_CLIENT_ID` and `AZURE_CLIENT_SECRET` set, and the `DefaultAzureCredential` is attempting to use `EnvironmentalCredentials` for an app that doesn't have access to the dev KV.
+When running locally:
```

Comment:
Maybe we need to reference instructions about local BAR from Maestro in case we're re-using it?

Reply:
I put a URL with the Maestro docs that have these instructions a few lines below